unity-renderer/Assets/DCLPlugins/ECS7/ECSComponents/VideoPlayer/VideoPlayerHandler.cs

```diff
@@ -89,7 +89,8 @@ public void OnComponentModelUpdated(IParcelScene scene, IDCLEntity entity, PBVid
             {
                 videoPlayer?.Dispose();
 
-                var id = entity.entityId.ToString();
+                var entityId = entity.entityId.ToString();
+                string id = scene.sceneData.sceneNumber > 0 ? scene.sceneData.sceneNumber + entityId : scene.GetHashCode() + entityId;
```

Comment:
I don't think we're using this variable at all, maybe I'm missing something?

Reply:
Its used on line 109

Reply:
Thanks, I was missing something 😆